MSCL/source/mscl/MicroStrain/Inertial/ExposedInertialTypes.h

```diff
@@ -2803,4 +2803,117 @@ namespace mscl
     //API Typedef: GpioPinOptions
     // A map of uint GPIO pin ID, <GpioFeatureBehaviors> pairs
     typedef std::map<uint8, GpioFeatureBehaviors> GpioPinOptions;
+
+    //API Struct: EventTriggerGpioParameter
+    struct EventTriggerGpioParameter
+    {
+        //API Enum: Mode
+        //  How the pin state affects the trigger
+        //      DISABLED   - 0x00 - The pin will have no effect and the trigger will never activate
+        //      WHILE_HIGH - 0x01 - The trigger will be active while the pin is high
+        //      WHILE_LOW  - 0x02 - The trigger will be active while the pin is low
+        //      EDGE       - 0x04 - Use if the pin is configured for time-stamping via the 3DM GPIO Configuration command (0x0C41)
+        enum Mode
+        {
+            DISABLED   = 0X00,  // The pin will have no effect and the trigger will never activate
+            WHILE_HIGH = 0X01,  // The trigger will be active while the pin is high
+            WHILE_LOW  = 0X02,  // The trigger will be active while the pin is low
+            EDGE       = 0X04   // Use if the pin is configured for time-stamping via the 3DM GPIO Configuration command (0x0C41)
+        };
+
+        // GPIO pin number
+        uint8 pin;
+
+        // GPIO pin mode
+        Mode mode;
+    };
+
+    //API Struct: EventTriggerThresholdParameter
+    struct EventTriggerThresholdParameter
+    {
+        //API Enum: Type
+        //  How the pin state affects the trigger
+        //      WINDOW_TYPE   - 0x01 - Window comparison
+        //      INTERVAL_TYPE - 0x02 - Trigger at evenly-spaced intervals
+        enum Type
+        {
+            WINDOW_TYPE   = 0x01, // Window comparison
+            INTERVAL_TYPE = 0x02  // Trigger at evenly-spaced intervals
+        };
+
+        // MIP channel field
+        MipTypes::ChannelField channelField;
+
+        // 1-based index of the target parameter within the MIP field
+        uint8 parameterId;
+
+        // Determines the type of comparison
+        Type type;
+
+        // Low threshold
+        double lowThreshold;
+        // High threshold
+        double highThreshold;
+    };
+
+    //API Struct: EventTriggerCombinationParameter
+    struct EventTriggerCombinationParameter
+    {
+        const uint16 LOGIC_NEVER   = 0x0000; // Never active
```

Comment:
Could you make these `static constexpr` instead of `const`, that plays more nicely with linux